src/js/shield_canvas_draw.js

```diff
@@ -189,16 +190,15 @@ export function trapezoid(
   outlineWidth,
   rectWidth
 ) {
-  let angleInRadians = (angle * Math.PI) / 180;
-  let angleSign = Math.sign(angle);
-  let sine = Math.sin(Math.abs(angleInRadians));
-  let cosine = Math.cos(angleInRadians);
-  let tangent = Math.tan(angleInRadians);
+  let angleSign = shortSideUp ? -1 : 1;
+  let sine = Math.sin(angle);
+  let cosine = Math.cos(angle);
+  let tangent = Math.tan(angle);
 
   if (rectWidth == null) {
     var shieldWidth =
       ShieldText.calculateTextWidth(ref, genericShieldFontSize) +
-      (2 + (CS * sine) / 2) * PXR;
+      (2 + ((CS / PXR) * tangent) / 2) * PXR;
```

Comment:
If I did this right, this simplifies to

```
2*PXR + CS * tangent / 2
```
Does that expression make sense for what this code is doing?  If so, that should be a bit cleaner to read without the parentheses and repeated terms PXR and 2.

Reply:
Makes sense, fixed.